Makefile

```diff
@@ -23,7 +23,7 @@ maintainer-clean: distclean ## Delete all non versioned files
 	rm -rf .pytest_cache
 	rm -rf tests/.pytest_cache
 	find . -name '*.orig' -delete
-	docker-compose down --remove-orphans --volumes --rmi all
+	docker compose down --remove-orphans --volumes --rmi all
```

Comment:
[About all of these changes](https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose). I know we've dealt with this in other repos by looking for both `docker compose` and `docker-compose` and putting that command into a variable, but maybe we should just force the switch here?

Reply:
I'm good with using `docker compose` without a variable yes ;)